astro/src/content/docs/extend/events-and-webhooks/events/user-login-failed.mdx

```diff
@@ -47,6 +47,22 @@ export const eventType = 'user.login.failed';
     Moved to <InlineField>event.info</InlineField> in 1.30.0
   </APIField>
 
+  <APIField slot="trailing-fields" name="event.reason" type="Object" since="1.53.0">
+    An object containing data on the reason the login failed if it was prevented by a [Login Validation Lambda](/docs/extend/code/lambdas/login-validation).
+  </APIField>
+
+  <APIField slot="trailing-fields" name="event.reason.code" type="String" since="1.53.0">
+    The theme message key used to find a message to display to the end user.
+  </APIField>
+
+  <APIField slot="trailing-fields" name="event.reason.lambdaId" type="UUID" since="1.53.0">
+    The lambda Id which triggered the login failure.
```

Comment:
This field is only present if the `reason.code` is `lambdaValidation`. It may be more correct to say this is the unique Id of the lambda that caused the login to fail?